LumpNav/LumpNav.py

```diff
@@ -706,6 +762,45 @@ def setupNavigationPanel(self):
     self.deleteLastFiducialDuringNavigationButton.setEnabled(False)
     self.contourAdjustmentFormLayout.addRow(self.deleteLastFiducialDuringNavigationButton)
 
+    # Distance Collapsible
+    self.distanceCollapsibleButton = ctk.ctkCollapsibleGroupBox()
+    self.distanceCollapsibleButton.title = "Distance"
+    self.distanceCollapsibleButton.collapsed=True
+    self.navigationCollapsibleLayout.addRow(self.distanceCollapsibleButton)
+
+    # Layout within the collapsible button
+    self.distanceFormLayout = qt.QFormLayout(self.distanceCollapsibleButton)        
+
+    self.activateDistanceButton = qt.QPushButton("Activate")
+    setButtonStyle(self.activateDistanceButton)
+    self.activateDistanceButton.setCheckable(True)
+    self.distanceFormLayout.addRow(self.activateDistanceButton)
+    
+    self.distanceComboBoxLabel = qt.QLabel('Text location:')

```

Comment:
Do you mean it is too much detail visible to the user, i.e., text position and visibility of crosshair and trajectory? 

Could you maybe elaborate on what you meant by "Would it be possible to figure this out automatically?"

With configurable through the .ini file, do you mean by just setting a True/False parameters in that file and not even include it in the GUI?